recipes/podofo/all/conanfile.py

```diff
@@ -0,0 +1,91 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class OatppSwaggerConan(ConanFile):
+    name = "podofo"
+    license = "GNU-2"
+    homepage = "http://podofo.sourceforge.net"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "PoDoFo is a library to work with the PDF file format."
+    topics = ("conan", "PDF", "PoDoFo", "podofo")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake", "cmake_find_package"
+    exports_sources = "CMakeLists.txt"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+        if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) < "5":
+            raise ConanInvalidConfiguration("PoDoFo requires GCC >=5")
+
+    def requirements(self):
+        self.requires("freetype/2.10.4")
+        if self.settings.os == "Linux":
```

Comment:
```suggestion
        if str(self.settings.os) in ["Linux", "Macos"]:
```
This would be a first step to solve macos error